src/crypto/local.js

```diff
@@ -141,8 +198,11 @@ function compressPoint(P) {
  */
 function decompressPoint(bytes) {
     const yTag = bytes[0];
+    const expLength = Math.ceil(sjcl.bitArray.bitLength(CURVE.r.toBits())/8)+1;
```

Comment:
Is there a reason for not simply using CURVE.r.bitLength()/8 + 1?

Reply:
Good point, we can just bitlength on the sjcl.bn type, I hadn't noticed that. In terms of the `+1` vs `Math.ceil`, I think Math.ceil is a bit more idiomatic and makes it clear that the result of the division may be a non-integer

Reply:
Actually Math.ceil is required because we need to get the number of bytes + 1

Reply:
I don't think so. From what I can see, is that bitLength() returns number of bits, but roudned up to nearest byte. So Math.ceil is useless.
You need +1 because expectedLength is a length of prime order + 1 byte for "compression flag"

Reply:
Ah yep, you're correct, it is useless!